data/FilterList.json

```diff
@@ -16830,5 +16830,29 @@
     "name": "AdHell Standart",
```

Comment:
mind fixing the spelling error to "Standard" from your last PR? thanks.

Reply:
I thought that there are other lists, something like "Full" or something like that, well, okay